modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/CreateClientExtensionConfigTask.java

```diff
@@ -203,8 +205,19 @@ private Provider<RegularFile> _addTaskOutputFile(String path) {
 		DirectoryProperty buildDirectoryProperty =
 			projectLayout.getBuildDirectory();
 
+		Provider<Directory> clientExtensionBuildDirProvider =
+			buildDirectoryProperty.dir(
+				ClientExtensionProjectConfigurator.CLIENT_EXTENSION_BUILD_DIR);
+
+		Directory clientExtensionBuildDir =
+			clientExtensionBuildDirProvider.get();
+
+		RegularFile regularFile = clientExtensionBuildDir.file(path);
```

Comment:
Ok i think we don't want to short circuit gradle here and instead we want to keep things late -binding.  And I think the way to do that is to create a new directoryProperty on our task that sets the clientExtensionBuild directory and it can be done the same way.